packages/odyssey-react/src/components/Infobox/index.tsx

```diff
@@ -0,0 +1,110 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import React from 'react';
+import type { FunctionComponent, ReactNode } from 'react';
+import { useCx, useOmit } from '../../utils';
+
+export type InfoboxVariants = 'info' | 'danger' | 'caution' | 'success';
+
+export type Props = {
+  /**
+   * Content to be rendered within the infobox. Avoid using direct children, put child content
+   * within the provided Infobox static components (Infobox.Content and Infobox.Actions)
```

Comment:
If this is the intended useage we should enforce (what we can) through type annotations by using `ReactElement | ReactElement[]` over `ReactNode`

Reply:
you can write tests against this behavior to now that you've updated it [see this code here.](https://github.com/okta/odyssey/pull/1001/files#diff-7e13dc6a75aab8f3e004a57efb75ab66c6ccaea00a7dd7fc3db6cd8655d985b6R47-R56)